pabi_procurement/wizard/print_purchase_work_acceptance.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, _, exceptions
```

Comment:
Better to use,
`
form openerp.exceptions import Warning as UserError
`
Then, use UserError instead of Warning in the code.